aggregator/.env.example

```diff
@@ -11,13 +11,8 @@ PRIVATE_KEY_AGG=0x5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab36
 PRIVATE_KEY_ADMIN=0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d
 TEST_BLS_WALLETS_SECRET=test-bls-wallets-secret
 
-PG_HOST=127.0.0.1
-PG_PORT=5432
-PG_USER=bls
-PG_PASSWORD=generate-a-strong-password
-PG_DB_NAME=bls_aggregator
+DB_PATH=aggregator.db
 
-BUNDLE_TABLE_NAME=bundles
```

Comment:
The idea of setting the table name was two-fold:
- Different table names for different networks
- Use throwaway names when testing

However, with sqlite it makes more sense to just use a different file for different networks. In testing we can do even better and just avoid the filesystem completely by using sqlite's in-memory feature.